frontend/public/components/app.jsx

```diff
@@ -155,13 +163,36 @@ class App extends React.PureComponent {
             </ConnectedNotificationDrawer>
           </Page>
           <CloudShell />
+          <GuidedTour />
           <ConsoleNotifier location="BannerBottom" />
         </GuidedTourDrawer>
+      </>
+    );
+
+    return (
+      <DetectPerspective>
+        {/* ToDo: contribute this context through extension */}
+        <Tour>
+          {contextProviderExtensions.reduce(
+            (children, provider) => (
+              <EnhancedProvider {...provider.properties}>{children}</EnhancedProvider>
+            ),
+            content,
+          )}
+        </Tour>
```

Comment:
Working on this. While using the TourContext through extension breaks the UI. Error:
```
job.js:17 Uncaught ReferenceError: Cannot access 'allModels' before initialization
    at Module.allModels (job.js:17)
    at Module.<anonymous> (ui.ts:106)
    at Module../public/actions/ui.ts (ui.ts:371)
    at __webpack_require__ (bootstrap:878)
    at fn (bootstrap:150)
    at Module.<anonymous> (ui.ts:1)
    at Module../public/reducers/ui.ts (ui.ts:395)
    at __webpack_require__ (bootstrap:878)
    at fn (bootstrap:150)
    at Module.<anonymous> (StepHeader.tsx:10)
```